SVG.flood/programs/animations.flp

```diff
@@ -58,6 +58,26 @@ func shadowAnimations(timeline, layer, id)
     return animations(properties: shadowProperties(), layer: layer, timeline: timeline, hrefID: "#\(id)-shadow-filter", attributeType: "XML")
 end
 
+func fillImageAnimation(layer, timeline)
+    var track = timeline.trackFor(layer: layer, property: "fillImage")
+    var keytimes = keyTimesFrom(track: track)
+    var keysplines = keysplinesFrom(track: track)
+
```

Comment:
lets put:
`var values = hrefValuesFrom(track: track, layer: layer).joined(separator: ";")`

Reply:
for consistency

Reply:
can we do: 
`var values = valuesFrom(track, layer)`

Reply:
Remove this function